docs/_includes/footer.md

```diff
@@ -1,6 +1,6 @@
 <footer>
   <p class="light tiny align_center">
-    © 2020 Slack Technologies, Inc. and contributors
+    © 2020- Slack Technologies, LLC and contributors
```

Comment:
I'm uncertain whether or not the inclusion of "and contributors" is OK, but I'm pretty unfamiliar with the legal side of things in this space.